pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -858,6 +858,16 @@ self: super: builtins.intersectAttrs super {
     );
   hercules-ci-cnix-store = super.hercules-ci-cnix-store.override { nix = pkgs.nixVersions.nix_2_9; };
 
+  # the testsuite fails because of not finding tsc without some help
+  aeson-typescript = overrideCabal (drv: {
+    buildTools = drv.buildTools or [] ++ [ pkgs.nodePackages.typescript ];
```

Comment:
One nitpick: I think this should be `testToolDepends` instead of `buildTools`.

If you look through `pkgs/development/haskell-modules/generic-builder.nix` you can see that `testToolDepends` is used almost the same as `buildTools`, except that it gets ignored if you set `dontCheck` (which I believe is what we want in this case).

Other than that, this LGTM, thanks!

Reply:
oh, brilliant, i was looking for that but missed it. Will change.